resources/views/profile/show.blade.php

```diff
@@ -2,7 +2,9 @@
     <div class="flex flex-col items-center justify-center py-10">
         <div class="min-h-screen w-full max-w-md overflow-hidden rounded-lg px-4 dark:shadow-md md:px-0">
             <livewire:links.index :userId="$user->id" />
-            <livewire:questions.create :toId="$user->id" />
+            @if($user->prefers_questions)
+                <livewire:questions.create :toId="$user->id" />
```

Comment:
Make sure the user can share an update even if the user is not wanting to receive questions.

Reply:
I am doing an user check if the user is the same as logged in user it will show the form.